kiosk/src/Models/Kiosk.ts

```diff
@@ -72,6 +74,14 @@ export class Kiosk {
         this.intervalId = setInterval(() => this.gamePadLoop(), configData.GamepadPollLoopMilli);
 
         window.addEventListener("message", (event) => {
+            if (event.data?.js) {
+                let builtGame: BuiltSimJSInfo | void = this.getBuiltGame(this.launchedGame);
```

Comment:
```suggestion
                let builtGame: BuiltSimJSInfo | undefined = this.getBuiltGame(this.launchedGame);
```

void type is not typically used besides return when a value is not to be read~